doc/authentication-methods/http.md

```diff
@@ -1,39 +1,53 @@
 ## Overview
 
-The purpose of this module is to connect with an external REST API and delegate the authentication operations to it whenever possible.
-The component must implement the API described in one of the next sections for `ejabberd_auth_http` to work out of the box.
+The purpose of this method is to connect with an external REST API and delegate the authentication operations to it whenever possible.
+The component must implement the API described in one of the next sections to work out of the box.
```

Comment:
I think it would be appropriate to link to the corresponding section, instead of saying "one of the next sections" 😅